docs/source/user-guide/configuration/settings.rst

```diff
@@ -64,6 +64,33 @@ repository, an administrator can set both :ref:`channel alias <channel-alias>` a
 
 .. _auto-update-conda:
 
+
+.. _channel-settings:
+
+``channel_settings``: Extra settings for individual channels
+------------------------------------------------------------
+
+With ``channel_settings``, it is possible to add extra configuration options
```

Comment:
```suggestion

.. versionadded:: 23.3.0

With ``channel_settings``, it is possible to add extra configuration options
```

This should render a nice version notification, commonly used in Sphinx projects, https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded

Reply:
Good idea!

Reply:
(we're sadly not very consistent with this)

Reply:
This whole page needs actually some work 🙃. One step at a time.